iothub/service/src/Amqp/AmqpClientHelper.cs

```diff
@@ -13,11 +13,14 @@
 
 namespace Microsoft.Azure.Devices.Amqp
 {
-    internal class AmqpClientHelper
+    /// <summary>
+    /// Miscellaneous helpers for AMQP operations.
+    /// </summary>
+    internal static class AmqpClientHelper
     {
-        public delegate object ParseFunc<in T>(AmqpMessage amqpMessage, T data);
+        internal delegate object ParseFunc<in T>(AmqpMessage amqpMessage, T data);
```

Comment:
internal classes should have internal (not public) types when possible. When the internal class inherits from a public class or interface, those inherited members must be public.